TourMate/TourMate/Frontend/Plan/Views/AddPlanViews/AddPlanViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AddPlanViewModel.swift
+//  TourMate
+//
+//  Created by Tan Rui Quan on 19/3/22.
+//
+
+import Foundation
+
+@MainActor
+class AddPlanViewModel: ObservableObject {
```

Comment:
This file is not present in my local branch but pushing and pulling doesn't seem to work. Git says that "everything is up to date"

Reply:
Can u repull? I think u deleted it here: https://github.com/Vielheim/TourMate/pull/81/files#diff-8b26f1f90b22c716f4f658535846ea931d6f5ffe049e86258b0040aed427df16

Reply:
i get 'already up to date' when i repull

Reply:
Actually why did u add this back? I thought that we are now using `AddPlanFormViewModel`

Reply:
I don't recall adding it back tho :/

Reply:
It's probably because its not detected by xcode, i think we can just delete it